itest/list_on_test.go

```diff
@@ -531,4 +531,8 @@ var allTestCases = []*lntest.TestCase{
 		Name:     "htlc timeout resolver extract preimage local",
 		TestFunc: testHtlcTimeoutResolverExtractPreimageLocal,
 	},
+	{
+		Name:     "chainkit",
+		TestFunc: testChainKit,
```

Comment:
Is there a reason this test is excluded from `list_on_test.go`? I thought this was an appropriate spot for the new test.

Reply:
It is, see L226.

Reply:
I missed that it is already listed. Removed this change.